src/scene/vcMedia.cpp

```diff
@@ -144,28 +171,41 @@ void vcMedia::AddToScene(vcState *pProgramState, vcRenderData *pRenderData)
   if (!m_visible)
     return;
 
-  if (m_image.pTexture != nullptr)
+  if (m_image.pTexture != nullptr && m_loadStatus == vcSLS_Loaded)
   {
-    // For now brute force sorting (n^2)
-    double distToCameraSqr = udMagSq3(m_image.position - pProgramState->pCamera->position);
-    size_t i = 0;
-    for (; i < pRenderData->images.length; ++i)
+    if (m_image.type == vcIT_StandardPhoto || m_image.type == vcIT_OrientedPhoto)
```

Comment:
Oriented Photo can go through the poly pipe as well

Reply:
That'd be the same poly model as a billboard right? I'll add them together in future PR

Reply:
Billboard is different, Oriented is just a normal mesh, Billboard needs to be expanded in screen space.

Reply:
same model different scale though right?

Reply:
Shouldn't be. It might be currently but that would be wrong.

The oriented model should be a 0-1 square and the billboard should be all positions at {0,0,0} and corners to say which way to expand.